pages/_app.tsx

```diff
@@ -115,25 +116,27 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
       >
         <DefaultSeoWithLocale />
         <ThemeProvider disableSmoothScrolling={isSearchOpen ? true : disableSmoothScrolling} headComponent={Head}>
-          <div
-            sx={{
-              position: 'absolute',
-              left: 0,
-              right: 0,
-              top: 0,
-              minHeight: '768px',
-              backgroundImage: `url('${process.env.BASE_PATH}/img/page-background.png')`,
-              backgroundSize: 'cover',
-              backgroundPosition: 'center top',
-              '@media (min-width: 1440px)': {
+          <div sx={{ position: 'absolute', left: 0, right: 0, top: 0, bottom: 0, overflow: 'hidden' }}>
```

Comment:
```suggestion
          <div sx={{ position: 'absolute', inset: 0, overflow: 'hidden' }}>
```

Reply:
Perfect, thank you. Have updated this. :)